.vscode/extensions.json

```diff
@@ -0,0 +1,15 @@
+{
+	// See https://go.microsoft.com/fwlink/?LinkId=827846 to learn about workspace recommendations.
+	// Extension identifier format: ${publisher}.${name}. Example: vscode.csharp
+	// List of extensions which should be recommended for users of this workspace.
+	"recommendations": [
+		"ms-python.vscode-pylance",
```

Comment:
Maybe instead of just pylance we could use the `ms-python.python` instead? Which comes with everything related to python, including pylance.

Reply:
Updated!